src/app/query/bitcoin/bitcoin-client.ts

```diff
@@ -51,7 +51,45 @@ export interface BestinslotInscriptionsByTxIdResponse {
   blockHeight: number;
 }
 
-class BestinslotInscriptionsApi {
+interface Brc20TokenResponse {
+  ticker: string;
+  overall_balance: string;
+  available_balance: string;
+  transferrable_balance: string;
+  image_url: string | null;
+}
+
+export interface Brc20Token extends Brc20TokenResponse {
+  decimals: number;
+  holderAddress: string;
+}
+
+interface Brc20TokenTicker {
```

Comment:
Should this also have `Response` in the name?

Reply:
hm, there is already in the file
```
interface Brc20TickerResponse {
  data: Brc20TokenTicker;
  block_height: number;
}
```